web/src/pages/HomePage/HomePage.tsx

```diff
@@ -108,47 +110,90 @@ const HomePage = () => {
           <h3>How it Works</h3>
           <ol>
             <li>
+              <h4>Register</h4>
               <a
                 href={Constants.APPLICATION_URL}
                 target="_blank"
                 rel="noreferrer"
               >
-                Register for the competition by 11:59pm PT, August 16, 2023.
+                Register to participate.
               </a>{' '}
-              This includes a short video (roughly 3 minutes or less),
-              introducing yourself, explaining your ideas, how you will use
-              RedwoodJS, and anything else you might want us to know.
+              You'll receive:
+              <ul>
+                <li>An invitation to a weekly virtual meetup (optional)</li>
+                <li>Access to a private Discord channel</li>
+                <li>Connect with other participants and build a team</li>
+                <li>FREE Sticker</li>
+                <li>
+                  Entry to our raffle and win 2{' '}
+                  <a
+                    href={Constants.CONFERENCE_URL}
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    RedwoodJS Conference Tickets
+                  </a>
+                </li>
+              </ul>
             </li>
             <li>
+              <h4>Build</h4>
               You&apos;ll have until{' '}
-              <strong>11:59pm PT September 8, 2023</strong> to build your
-              project.
+              <strong className="text-carminePink">
+                11:59pm PT September 8, 2023
+              </strong>{' '}
+              to build your project.
+            </li>
+            <li>
+              <h4>Submit</h4>
+              <strong className="text-carminePink">
+                Submit your final project
+              </strong>{' '}
+              for review.
+            </li>
+          </ol>
+        </section>
+        <section>
+          <h3>Resources</h3>
+          <ul>
+            <li>
+              <a href={Constants.TUTORIAL} target="_blank" rel="noreferrer">
+                Quick Start tutorial
+              </a>
             </li>
             <li>
-              Three finalists will be chosen to present their projects at the{' '}
               <a
-                href={Constants.CONFERENCE_URL}
+                href={Constants.REDWOODJS_URL}
                 target="_blank"
                 rel="noreferrer"
               >
-                RedwoodJS Conference
+                RedwoodJS Official Documentation
+              </a>
+            </li>
+            <li>
+              <a href={Constants.CONFERENCE_URL}>RedwoodJS Conference</a>
+            </li>
+            <li>
+              <a href={Constants.APP_IDEAS} target="_blank" rel="noreferrer">
+                Here are some app ideas
               </a>{' '}
-              in Grants Pass, Oregon, September 26- 29th.
+              to inspire you.
```

Comment:
```suggestion
              to inspire you. (coming soon!)
```

Just don't want to hold up launch!